language/stdlib/modules/LibraConfig.move

```diff
@@ -224,6 +224,26 @@ module LibraConfig {
 
         let config_ref = borrow_global_mut<Configuration>(CoreAddresses::LIBRA_ROOT_ADDRESS());
         let current_time = LibraTimestamp::now_microseconds();
+
+        // Do not do anything if a reconfiguration event is already emitted within this transaction.
+        //
+        // This is OK because:
+        // - The time changes in every non-empty block
+        // - A block automatically ends after a transaction that emits a reconfiguration event, which is guarenteed by
+        //   LibraVM spec that all transactions comming after a reconfiguration transaction will be returned as Retry
+        //   status.
+        // - Each transaction must emit at most one reconfiguration event
+        //
+        // Thus, this check ensures that a transaction that does multiple "reconfiguration required" actions emits only
+        // one reconfiguration event.
+        //
+        // Another footnote is that the reasoning above hinders reconfiguration in a NIL block after a reconfiguration
```

Comment:
I don't understand this last part. What configuration changes are proposed in a block prologue?

Reply:
No configuration change is currently in the block prologue. Writing this down just to remind us that we couldn't do that in the future

Reply:
I think I would leave this out--for me it raises more questions than it answers.